judge/forms.py

```diff
@@ -254,6 +256,30 @@ def clean(self):
             raise ValidationError(_('Must specify either totp_token or webauthn_response.'))
 
 
+class WCIPEGMergeActivationForm(Form):
+    accept = BooleanField(label=_('I understand and would still like to merge my accounts:'))
+
+
+class WCIPEGMergeRequestForm(Form):
+    handle = CharField(max_length=50, validators=[UnicodeUsernameValidator()])
+
+    if ReCaptchaField is not None:
+        captcha = ReCaptchaField(widget=ReCaptchaWidget())
+
+    def clean_handle(self):
+        try:
+            profile = Profile.objects.get(user__username=self.cleaned_data['handle'], user__is_active=True)
+            if profile.is_peg:
```

Comment:
We should probably move this `if` into the `else` branch since it's more "Pythonic".